app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -100,7 +100,7 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           :data => {'toggle'        => 'modal',
                     'target'        => '#modal_d_div',
                     'function'      => 'miqCallAngular',
-                    'function-data' => '{"name": "showListener", "args": []}'})
+                    'function-data' => '{\"name\": \"showListener\", \"args\": []}'})
```

Comment:
When this string was not escaped and passed to html, then `JSON.parse()` got confused because the string looked like this:
```JSON
{
"function-data": "{"name": "showListener", "args": []}"
}
```

Hence the need of escapig:
```JSON
{
"function-data": "{\"name\": \"showListener\", \"args\": []}"
}
```